crates/bevy_window/src/window.rs

```diff
@@ -107,7 +118,7 @@ pub struct Window {
     pub title: String,
     /// How the alpha channel of textures should be handled while compositing.
     pub composite_alpha_mode: CompositeAlphaMode,
-    /// Which size limits to give the window.
+    /// The size limits the window's resolution should not exceed.
```

Comment:
I don't really like the "resolution" terminology for what I suspect is just the window size? "resolution" feels like it could refer also to DPI for example, and is generally only used in the context of monitor/screen resolution.

Reply:
The "problem" I have is that the size is inside the `WindowResolution` struct.

I'm not sure how to communicate both that it's the window's size, and where to find it.

Proposed change:

```rust
The size limits the window's physical size (found in its [`resolution`](WindowResolution) should not exceed.
```
?

Reply:
Maybe resolved by https://github.com/bevyengine/bevy/pull/8858/commits/a36128c631ab0677c3cfd13e09ff2026af4b5315 ?

I ended up with:
```rust
    /// The limits of the window's physical size
    /// (found in its [`resolution`](WindowResolution) when resizing.
```

Reply:
Ok that makes sense. I like the solution, thanks!